starters/express-typescript-typeorm-postgres/src/db/datasource.ts

```diff
@@ -12,6 +11,33 @@ export const dataSource = new DataSource({
   password: process.env.DATABASE_PASSWORD,
   database: process.env.DATABASE_NAME,
   entities: ['**/*.entity.js'],
-  logging: IS_DATABASE_LOGGING_ENABLED,
-  synchronize: true,
+  logging: DATABASE_ENABLE_LOGGING,
+  synchronize: DATABASE_ENABLE_SYNC,
 });
+
+const DATABASE_RETRY_COUNT = process.env.DATABASE_RETRY_COUNT
+  ? parseInt(process.env.DATABASE_RETRY_COUNT)
+  : 25;
```

Comment:
`25` retries by default?  Isn't that a lot?

Reply:
I was testing something and forgot it there, I updated it to be .env based and updated the .env.example file accordingly.